ENIGMAsystem/SHELL/Universal_System/Instances/with.h

```diff
@@ -1,39 +1,39 @@
-**  applications created by its users, or damages caused by the environment     **
-**  or programs made in the environment.                                        **
-**                                                                              **
-\********************************************************************************/
+// /********************************************************************************\
+// **                                                                              **
+// **  Copyright (C) 2008 Josh Ventura                                             **
+// **  Copyright (C) 2014 Seth N. Hetu                                             **
+// **                                                                              **
+// **  This file is a part of the ENIGMA Development Environment.                  **
+// **                                                                              **
+// **                                                                              **
+// **  ENIGMA is free software: you can redistribute it and/or modify it under the **
+// **  terms of the GNU General Public License as published by the Free Software   **
+// **  Foundation, version 3 of the license or any later version.                  **
+// **                                                                              **
+// **  This application and its source code is distributed AS-IS, WITHOUT ANY      **
+// **  WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS   **
+// **  FOR A PARTICULAR PURPOSE. See the GNU General Public License for more       **
+// **  details.                                                                    **
+// **                                                                              **
+// **  You should have recieved a copy of the GNU General Public License along     **
+// **  with this code. If not, see <http://www.gnu.org/licenses/>                  **
+// **                                                                              **
+// **  ENIGMA is an environment designed to create games and other programs with a **
+// **  high-level, fully compilable language. Developers of ENIGMA or anything     **
+// **  associated with ENIGMA are in no way responsible for its users or           **
+// **  applications created by its users, or damages caused by the environment     **
+// **  or programs made in the environment.                                        **
+// **                                                                              **
+// \********************************************************************************/
 
-#define with(x) \
-  for (enigma::iterator::with with(enigma::fetch_inst_iter_by_int(x)); \
-      enigma::instance_event_iterator; \
-      enigma::instance_event_iterator = enigma::instance_event_iterator->next)
+// #define with(x) \
```

Comment:
It is now defined [here](https://github.com/enigma-dev/enigma-dev/blob/dad033e022ea334f85b569919eba334c123c11c3/CompilerSource/compiler/components/write_object_data.cpp#L858-L861).